src/main/java/com/dsl/clauses/LimitClause.java

```diff
@@ -0,0 +1,16 @@
+package com.dsl.clauses;
+
+import com.dsl.AsString;
+
+public class LimitClause implements AsString, Clause {
+  private final int numElements;
+
+  public LimitClause(final int numElements) {
```

Comment:
The limit clause allows at least a variable as input, so in this case, com.dsl.expressions.param.Variable should be a possible constructor argument. Also, there's the possibility of having an expression on the LIMIT clause like in https://neo4j.com/docs/cypher-manual/current/clauses/limit/#limit-subset-rows-using-expression

Reply:
Could you help to test the constructor with expression? I don't have a clue how to create such expression.

Reply:
Will do